### PR TITLE
Add AeroNav RFD charts

### PR DESCRIPTION
**Pull request by awdev1**
# pls read carefully thx
# git was weird but it seems fine (you might wanna check it before we deploy it at all 💀idk)
- Add AeroNav hopefully without a krill issue
- Forced @aidenbok203 against his will to look at the charts to verify it being correct (joking on the against part)
- attempted to match the chart categories to match chart fox (dm me if its wrong)
- the thing telling you that the gates.png in ILAR was removed is incorrect:
- 
image
### hmm most likely caus i formatted that part
image
- did not put runway numbers in STARS and SIDS as it just made a BIG mess

- thanks formic!

### CODE REVIEW

**Comment by aidenbok203** [comment]
yes

File: src/Charts.ts [hunk 1]
```diff
@@ -3224,13 +3224,212 @@ let charts: Chart[] = [
...
+        runways: [],
+    },
+    {
+        airport: "IRFD",
+        author: "AeroNav",
+        id: 356,
+        category: "SID",
+        name: "DARRK 3 RNAV DEPARTURE",
+        file: '/charts/rockford/AreoNav/SID/DARRK3.png',
+        runways: [],
```
Comment (FormicAcidGD) [inline src/Charts.ts]: Add runways

File: src/Charts.ts [hunk 2]
```diff
@@ -3224,13 +3224,212 @@ let charts: Chart[] = [
...
+        name: "AIRPORT BRIEFING 3",
+        file: '/charts/rockford/AreoNav/GEN/port3.png',
+        runways: [],
+    },
+    {
+        airport: "IRFD",
+        author: "AeroNav",
+        id: 368,
+        category: "APP",
+        name: "RNAV (RNP) RWY 7C",
```
Comment (FormicAcidGD) [inline src/Charts.ts]: Change Runway Numbers to European ones (07C instead of 7C)

Comment (awdev1) [reply src/Charts.ts]: done

File: src/Charts.ts (hunk 1 above)
Comment (awdev1) [reply src/Charts.ts]: no

**Comment by awdev1** [comment]
yayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy

**Pull request merged**